api/serializers/canteen.py

```diff
@@ -206,6 +209,26 @@ def create(self, validated_data):
 
         return canteen
 
+    def get_central_kitchen_diagnostics(self, obj):
+        if not obj.central_producer_siret:
```

Comment:
Question : should we also check that the `obj` is a `site_cooked_elsewhere` canteen ?